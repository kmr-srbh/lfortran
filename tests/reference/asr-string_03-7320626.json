{
    "basename": "asr-string_03-7320626",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/string_03.f90",
    "infile_hash": "7815b8703dfd3a1c61cab40604e20693508878e8a90e450834ce592b",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-string_03-7320626.stdout",
    "stdout_hash": "a88675cd5be17e5f3e45d448c5cfa0d850b4984961a799271f9635ed",
    "stderr": "asr-string_03-7320626.stderr",
    "stderr_hash": "20f1e15b0f0ca6a80217361f2543d3d905711ae0ccbb3816ab9dcf3f",
    "returncode": 0
}