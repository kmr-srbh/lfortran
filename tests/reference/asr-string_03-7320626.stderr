warning: The LHS character len=29 and the RHS character len=30 are not equal.
  --> tests/../integration_tests/string_03.f90:21:1
   |
21 | combined = intro//verb//posit//title//last_name//"."
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: consider changing the RHS character len to match the LHS character len
